### PR TITLE
Config slack auth0

### PR DESCRIPTION
**Pull request by MathiasMoralesDixtra**
- Created model to auth
- Added functionality to handle auth

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -15,11 +20,20 @@ const router = buildAdminRouter(adminBro);
 // Create a Bolt Receiver
 const receiver = new ExpressReceiver({
   signingSecret: process.env.SLACK_SIGNING_SECRET,
+  clientId: process.env.SLACK_CLIENT_ID,
```
Comment (mauriguz) [inline app.js:23]: did you add these to secrets?

File: controllers/auth.controller.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+const { Auth } = require('../entities/auth.entities');
+
+const storeInstallation = async (installation) => {
+  if (installation.team !== undefined) {
```
Comment (mauriguz) [inline controllers/auth.controller.js:4]: is it possible that installation.team equals to null?

File: entities/auth.entities.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    bot: {
+      scopes: [String],
+      token: { type: String },
+      userId: { type: String },
+      id: { type: String },
+    },
+    incomingWebhook: {
+      url: { type: String },
+      channel: { type: String },
+      channelId: { type: String },
```
Comment (mauriguz) [inline entities/auth.entities.js:31]: what's the difference between `channel` and `channelId`?

File: app.js (hunk 1 above)
Comment (MathiasMoralesDixtra) [reply app.js:23]: I'm going to add them

File: controllers/auth.controller.js (hunk 2 above)
Comment (MathiasMoralesDixtra) [reply controllers/auth.controller.js:4]: This validation is provided by the documentation so I suppose that it returns a value or undefined.

File: entities/auth.entities.js (hunk 3 above)
Comment (MathiasMoralesDixtra) [reply entities/auth.entities.js:31]: the channel is the name of the channel and channeled is the id.

File: app.js (hunk 1 above)
Comment (MathiasMoralesDixtra) [reply app.js:23]: Done

File: controllers/auth.controller.js (hunk 2 above)
Comment (mauriguz) [reply controllers/auth.controller.js:4]: ok

**Review by mauriguz — APPROVED** [review]

**Pull request merged**